src/components/SignList.js

```diff
@@ -0,0 +1,47 @@
+import React from "react";
+import {
+  ListGroup,
+  ListGroupItem,
+  ListGroupItemText,
+  CardImg,
+} from "shards-react";
+import classNames from "classnames";
+import Store from "../flux/store";
+import images from "../images/mfd/sign-sample/index";
+
+const SignList = ({ filter }) => {
+  const classes = classNames("mt-3");
+
+  return (
+    <div className={classes}>
+      {Store.getSignItems()
+        // eslint-disable-next-line array-callback-return
+        .filter((val) => {
+          if (filter === "") return val;
```

Comment:
Let's add the open close braces even it is only one line of code to execute after it. This is to make the code easy to read and maintain.

Reply:
ALright. I will add that in an upcoming commit.